File: python/export.py
import requests

# get your API key https://app.ioannotator.com/api
params= {'apikey': 'add your API key here',
         'dataset': 'add your dataset ID here',}

api = 'https://api.ioannotator.com/export'


x = requests.get(api, params=params)

print(x.text)

File: r/text/export.R
library(httr)
library(rlist)
library(jsonlite)

# get your API key https://app.ioannotator.com/api
query = list(apikey = "add your API key here", dataset="add your dataset ID here")

res <- GET("https://api.ioannotator.com/export", query = query, verbose())
http_type(res)
jsonRespText<-content(res,as="text") 
fromJSON(jsonRespText)

